libs/solid/quiz/src/lib/components/start/start.component.html

```diff
@@ -123,6 +123,7 @@ <h2>Selbsttest</h2>
               *ngFor="let tag of tags"
               [value]="tag"
               (click)="c.toggleSelected(true)"
+              [ngClass]="tags.length > 6 ? 'mat-chip-small' : 'mat-chip-big'"
```

Comment:
kannst Du das noch "big" in "large" ändern? Das ist mehr konform mit XS, S, M, L, XL etc.